src/components/NewsBar.vue

```diff
@@ -0,0 +1,47 @@
+<template> <!-- Just a cool way to show our news -->
+    <div class="containerBar marquee" >
+        <h6 >{{newNews}}</h6>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        newNews: {
+            type: String,
+            required: true,
+            default: "if you are watching it, some error may has happened or its just a random text for test it. Use the 'newNews' prop for change me. We will show in this bar some useful stuff"
+        }
+
+    }
+
+}
+</script>
+
+<style>
+
+.marquee {
+    display:flex;
+    align-items: center;
+    font-size: 20px;
+    width: 100%;
+    height: 50px;
```

Comment:
Maybe we can use postcss to create some reference variables to follow the responsive way